### PR TITLE
Add the option to add the Jira ticket number inline within the commit…

### PR DESCRIPTION
**Pull request by hy0ug0**
… message

### CODE REVIEW

**Comment by hy0ug0** [comment]
Hi @bk201- I need this kind of feature for my project so if you're interested we can make it happened otherwise I will fork your project if it's ok with you 😄

**Comment by bk201-** [comment]
Hi! I thought of using cosmiconfig. It's a quite popular library. Both previous PR also might be solved by config.

**Comment by hy0ug0** [comment]
Ok I gonna use `cosmiconfig` instead, otherwise are you ok with the PR ?

**Comment by bk201-** [comment]
In general, yes. Please, wait. I'm adding prettier and more useful eslint rules for me. I'll let you know when I finish, it should take some minutes. Thanks

**Comment by hy0ug0** [comment]
I added a basic config using `cosmiconfig`, from there we gonna be able to add more configuration. Also if you're interested we could add a `.gitlabci.yml` for some automatic CI/CD.

File: package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
     "@typescript-eslint/eslint-plugin": "^2.25.0",
     "@typescript-eslint/parser": "^2.25.0",
     "ava": "^3.5.1",
+    "cosmiconfig": "^6.0.0",
```
[the remark this answers is not in the document]
Comment (hy0ug0) [reply package.json]: Done

**Comment by hy0ug0** [comment]
My bad we are in GitHub not GitLab so the `.gitlabci.yml` would be useless in this case.

**Pull request closed without merging**